### PR TITLE
Updates: explore the default project

### PR DESCRIPTION
**Pull request by yotamhc**
Edits related to the docs working group task

### CODE REVIEW

File: modules/developers-guide/pages/tutorials/explore-templates.adoc [hunk 1]
```diff
@@ -178,7 +178,7 @@ For now, you can continue to the next section.
 [[start-the-local-network]]
 == Start the local network
 
-Before you can build the default project, you need to connect to the {IC} network either running *locally* in your development environment or running *remotely* on a subnet that you can access.
+Before you can build the default project, you need to connect to the {IC} network either running *locally* in your development environment or running *remotely* (on-chain) on a subnet that you can access.
```
Comment (Dfinity-Bjoern) [inline modules/developers-guide/pages/tutorials/explore-templates.adoc:181]: This should be "local canister execution environment" vs. "Internet Computer/IC/...". There is no "IC network running locally".

**Review by Dfinity-Bjoern — COMMENTED** [review]
Please make another pass replacing the various terms for the replica run within dfx with "local canister execution environment". Also, I think the figure currently suggests that one has to start the local canister execution environment even when deploying to the IC directly, which does not seem correct.

**Review by Dfinity-Bjoern — APPROVED** [review]
Text looks good, please update the figure as discussed.

**Comment by yotamhc** [comment]
Figure updated as discussed

**Pull request merged**